tests/test_posixtestsuite.py

```diff
@@ -0,0 +1,3466 @@
+#               args = posix_tests_common_args + [
+#                   '-I', posix_test_dir('mmap')])
+
+
+  def test_mmap_24_2(self):
+    self.btest(posix_test_file('mmap', '24-2.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('mmap')])
+
+
+  def test_munmap_2_1(self):
+    self.btest(posix_test_file('munmap', '2-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('munmap')])
+
+
+  def test_munmap_4_1(self):
+    self.btest(posix_test_file('munmap', '4-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('munmap')])
+
+
+  def test_munmap_8_1(self):
+    self.btest(posix_test_file('munmap', '8-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('munmap')])
+
+
+  def test_munmap_9_1(self):
+    self.btest(posix_test_file('munmap', '9-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('munmap')])
+
+
+# Test fail: nanosleep() did not sleep long enough
+#  def test_nanosleep_1_1(self):
+#    self.btest(posix_test_file('nanosleep', '1-1.c'),
+#               expected = '0',
+#               args = posix_tests_common_args + [
+#                   '-I', posix_test_dir('nanosleep')])
+
+
+# Test fail: "At least one test FAILED"
+#  def test_nanosleep_2_1(self):
+#    self.btest(posix_test_file('nanosleep', '2-1.c'),
+#               expected = '0',
+#               args = posix_tests_common_args + [
+#                   '-I', posix_test_dir('nanosleep')])
+
+
+  def test_nanosleep_5_1(self):
+    self.btest(posix_test_file('nanosleep', '5-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('nanosleep')])
+
+
+  def test_nanosleep_6_1(self):
+    self.btest(posix_test_file('nanosleep', '6-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('nanosleep')])
+
+
+  def test_nanosleep_10000_1(self):
+    self.btest(posix_test_file('nanosleep', '10000-1.c'),
+               expected = '0',
+               args = posix_tests_common_args + [
+                   '-I', posix_test_dir('nanosleep')])
+
+
+# Test throws exception: need to fix thread function signature: "void*()" -> "void*(void*)"
+#  @requires_threads
+#  def test_pthread_attr_destroy_1_1(self):
+#    self.btest(posix_test_file('pthread_attr_destroy', '1-1.c'),
+#               expected = '0',
+#               args = posix_tests_common_args + [
+#                   '-s', 'USE_PTHREADS=1',
+#                   '-s', 'PTHREAD_POOL_SIZE=8',
+#                   '-I', posix_test_dir('pthread_attr_destroy')])
+
+
+  @requires_threads
```

Comment:
These test are marked as requires_threads but the above ones are not?

I wonder if we could run the other ones outside of browser tests?

For that matter, given that we now support node threads I wonder if we could make all of these just regular tests that run without the browser?